bibtex-actions-filenotify.el

```diff
@@ -0,0 +1,200 @@
+;;; bibtex-actions-filenotify.el --- filenotify functions for bibtex-actions -*- lexical-binding: t; -*-
+;;
+;; Copyright (C) 2021 Bruce D'Arcus
+;;
+;; Author: Bruce D'Arcus <https://github.com/bdarcus>
+;; Maintainer: Bruce D'Arcus <bdarcus@gmail.com>
+;; Created: August 17, 2021
+;; Modified: August 17, 2021
+;; Version: 0.0.1
+;; Keywords: abbrev bib c calendar comm convenience data docs emulations extensions faces files frames games hardware help hypermedia i18n internal languages lisp local maint mail matching mouse multimedia news outlines processes terminals tex tools unix vc wp
+;; Homepage: https://github.com/bruce/bibtex-actions-filenotify
+;; Package-Requires: ((emacs "24.3"))
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; Commentary:
+;;
+;;
+;;
+;;; Code:
+
+(require 'filenotify)
+(require 'files)
+(require 'bibtex-actions)
+
+(declare-function bibtex-actions-refresh "bibtex-actions")
+(declare-function bibtex-actions--local-files-to-cache "bibtex-actions")
+(declare-function bibtex-actions--global-files-to-cache "bibtex-actions")
+(declare-function bibtex-actions-utils--normalize-paths "bibtex-actions-utils")
+(declare-function reftex-access-scan-info "ext:reftex")
+(declare-function file-notify-add-watch "ext:file-notify")
+(declare-function file-notify-rm-watch "ext:file-notify")
+
+(defvar bibtex-actions-bibliography)
+(defvar bibtex-actions-library-paths)
+(defvar bibtex-actions-notes-paths)
+(defvar bibtex-actions-create-note-function)
+(defvar bibtex-actions-filenotify-files)
+(defvar bibtex-actions--candidates-cache)
+(defvar bibtex-actions--local-candidates-cache)
+(defvar bibtex-actions-filenotify-callback)
+(defvar bibtex-actions-open-file-function)
+(defvar bibtex-actions-file-extensions)
```

Comment:
```suggestion
```
`(require 'bibtex-actions)` should take care of these being defined and most aren't needed.